### ISSUE TITLE
Players reporting Character preferences not loading or long loading.

### ISSUE THREAD

**Issue by Titian3** [space-wizards/space-station-14#31128]
## Description
Players reporting Character preferences not loading or taking a long time to on joining.
"Your character preferences have not yet loaded"
No errors client side relating to it.

**Reproduction**
No idea yet, but multiple reports. will update if I find out.

**Screenshots**
![image](https://github.com/user-attachments/assets/a12063de-6aad-45d0-a7c8-07953a8991dd)

**Additional context**

**Comment by dragonryan06** [comment]
yeah i have no idea either but i saw that "please wait while they load" text for the first time ever within the last month so its something recent

**Comment by Titian3** [comment]
![image](https://github.com/user-attachments/assets/9c557ac8-a7d4-4283-9ac9-515fd1fd9b09)
It also send you to the void when it happens if you observe.

**Comment by UbaserB** [comment]
> yeah i have no idea either but i saw that "please wait while they load" text for the first time ever within the last month so its something recent

I've also briefly experienced this for a second or less, but it is also my first time seeing it

**Comment by FaustXD1** [comment]
Have you found a solution to the problem yet?
